I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 63, in script
    show mascot fixedmascot_size at right
Exception: Image 'mascot' does not accept attributes 'fixedmascot_size'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 63, in script
    show mascot fixedmascot_size at right
  File "C:\Users\justi\Desktop\renpy-8.2.0-sdk\renpy\ast.py", line 1113, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\justi\Desktop\renpy-8.2.0-sdk\renpy\ast.py", line 1076, in show_imspec
    renpy.config.show(name,
  File "C:\Users\justi\Desktop\renpy-8.2.0-sdk\renpy\exports.py", line 739, in show
    if not base.find_target() and renpy.config.missing_show:
  File "C:\Users\justi\Desktop\renpy-8.2.0-sdk\renpy\display\image.py", line 421, in find_target
    self.target = target._duplicate(a)
  File "C:\Users\justi\Desktop\renpy-8.2.0-sdk\renpy\display\displayable.py", line 246, in _duplicate
    args.extraneous()
  File "C:\Users\justi\Desktop\renpy-8.2.0-sdk\renpy\display\displayable.py", line 109, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'mascot' does not accept attributes 'fixedmascot_size'.

Windows-10-10.0.22631 AMD64
Ren'Py 8.2.0.24012702
The Hidden Secret 1.0
Sat May  4 01:59:23 2024
